library/registry

```diff
@@ -1,11 +1,13 @@
 # maintainer: Stephen Day <stephen.day@docker.com> (@stevvooe)
 # maintainer: Olivier Gambier <olivier.gambier@docker.com> (@dmp42)
 # maintainer: Richard Scothern <richard.scothern@gmail.com> (@RichardScothern)
+# maintainer: Aaron Lehmann <aaron.lehmann@docker.com> (@aaronlehmann)
 
 latest: git://github.com/docker/docker-registry@0.9.1
 0.8.1: git://github.com/docker/docker-registry@0.8.1
 0.9.1: git://github.com/docker/docker-registry@0.9.1
 
-2: git://github.com/docker/distribution@1341222284b3a6b4e77fb64571ad423ed58b0d34
+2: git://github.com/docker/distribution-library-image@d5ee820f89fc8ca7b66bd88f70554e89319808e0
 2.0: git://github.com/docker/distribution@1341222284b3a6b4e77fb64571ad423ed58b0d34
 2.0.1: git://github.com/docker/distribution@1341222284b3a6b4e77fb64571ad423ed58b0d34
```

Comment:
Can you please remove `2.0` and `2.0.1` from this file?  The already-built images stay up on the Hub, but since they're not going to receive further updates, removing them from here makes sure we don't spend time building them anew anymore and removes them from the "Supported" list in the image description.